src/SpecFlow.NetCore/Fixer.cs

```diff
@@ -61,7 +61,7 @@ public void Fix(DirectoryInfo directory)
 
 	    private void WarnNotExists(FileInfo featureFile)
 	    {
-	        WriteLine("New file generated: " + featureFile.FullName + ".cs. No tests in " + featureFile.Name + " will be discovered by dotnet test");
+	        WriteLine($@"New file generated: {featureFile.FullName}.cs. No tests in {featureFile.Name} will be discovered by dotnet test");
```

Comment:
Oops. Its a habit from my work project which uses style cop and style cop blows up on interpolation unless you use the marker. I'll fix it tomorrow, going to bed now.